### DISCUSSION TITLE
Why does 'x' then 'ctrl+a' not increment all numbers in line?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [helix-editor/helix#8097] [Q&A] [answered]
Title, I am used to shift+V and ctrl+a to increment all numbers under selection in vim. Any similar behavior in helix?

**Comment by the-mikedavis** [comment, marked as answer]
`C-a`/`C-x` act on the current selection when the whole selection is a number or date. You can select the line with `X` and then subselect the numbers with `s` using a regex like `\d+`, then hit Enter and use `C-a`/`C-x`.

**Reply by ntzm** [reply]
I had no idea this worked on dates, amazing
